galasa-extensions-parent/dev.galasa.ecosystem.builder/src/main/resources/scripts/docker-setup.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+source docker/config.properties
+
+echo "Pulling etcd"
+docker pull quay.io/coreos/etcd:v3.2.25
```

Comment:
dont need to pull.   the run will sort that out.   Also,  dont want to pull incase they have used a future "load" command to load the images